lms/static/sass/shared/_footer-edx.scss

```diff
@@ -7,17 +7,116 @@
 $edx-footer-link-color: $link-color;
 $edx-footer-bg-color: rgb(252,252,252);
 
-
 // Aggressively scoped for Drupal
 // ==============================
 // These styles are being loaded on Drupal, LMS and WordPress
 // sites so the scope has to be aggressive to override default
 // CMS styles
 footer#footer-edx-v3 {
   background: $edx-footer-bg-color;
-  padding: 20px;
+  padding: 15px 0;
   border-top: 1px solid $courseware-button-border-color;
+  font-family: "Open Sans" ;
+
+  div > div > div:nth-child(2) > div.col-lg-2.edx-col-md-2.col-md-3.col-sm-3 > ul {
+    border: none;
+  }
+
+
+  div > div.row.all-screens > div.col-lg-10.col-md-12.col-sm-9.col-xs-12 > div{
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    flex-wrap: wrap;
+
+  }
+  div > div.row.all-screens > div.col-lg-10.col-md-12.col-sm-9.col-xs-12 > div > div{
+    flex-direction: column;
+  }
+  div > div.row.all-screens > div.col-lg-10.col-md-12.col-sm-9.col-xs-12 > div > div.col-md-2.col-sm-4.col-xs-12 > div{
+    height: 100%;
+  }
+  div > div.row.all-screens > div.col-lg-10.col-md-12.col-sm-9.col-xs-12 > div > div.col-md-2.col-sm-4.col-xs-12 > div > div{
+    height: 100%;
+  }
+  div > div.row.all-screens > div.col-lg-10.col-md-12.col-sm-9.col-xs-12 > div > div.col-md-2.col-sm-4.col-xs-12 > div > div > ul{
+    height: 100%;
+  }
+  div > div.row.all-screens > div.col-lg-10.col-md-12.col-sm-9.col-xs-12 > div > div:nth-child(2) > div{
+    height: 100%;
+  }
+  div > div.row.all-screens > div.col-lg-10.col-md-12.col-sm-9.col-xs-12 > div > div:nth-child(2) > div > div{
+    height: 100%;
+  }
+  div > div.row.all-screens > div.col-lg-10.col-md-12.col-sm-9.col-xs-12 > div > div:nth-child(2) > div > div > ul{
+    height: 100%;
+  }
+
+  div > div.row.all-screens > div.col-lg-10.col-md-12.col-sm-9.col-xs-12 > div > div:nth-child(3) > div{
+    height: 100%;
+  }
+  div > div.row.all-screens > div.col-lg-10.col-md-12.col-sm-9.col-xs-12 > div > div:nth-child(3) > div > div{
+    height: 100%;
+  }
+  div > div.row.all-screens > div.col-lg-10.col-md-12.col-sm-9.col-xs-12 > div > div:nth-child(3) > div > div > ul{
+    height: 100%;
+  }
+  div > div.row.all-screens > div.col-lg-10.col-md-12.col-sm-9.col-xs-12 > div > div.col-md-4.col-sm-9.col-xs-12{
+    float: none;
+    display: table-cell;
+    vertical-align: top;
+  }
+
+  div.container{
+    min-width: 0;
+    @include padding(0, 15px, 0, 15px)
```

Comment:
This doesn't need to use the padding function as that is only necessary if the left and right values are different. You should also use ``$baseline``, so something like this:

    padding: 0 $baseline*0.75;